CHANGELOG.md

```diff
@@ -16,6 +16,12 @@
 * RealmCollection and OrderedRealmCollection have been added. RealmList and RealmResults both implement these interfaces.
 * RealmBaseAdapter now accept an OrderedRealmCollection instead of only RealmResults.
 * RealmObjectSchema.isPrimaryKey(String) (#2440)
+* RealmConfiguration.initialData() can now be used to populate a Realm file before it is used for the first time.
+
+### Credits
```

Comment:
Tried it. Looks a bit strange, so I think it is better to keep the current nesting. `###` looks really big though so we should probably consider switching all subsections to `####` to make it a bit easier on the eye.